assets/logos/beamery.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+const BeameryLogo = () => (
```

Comment:
ok i think this needs to be agreed by all of us. because in my PR i have now put it in public/assets/folder... 
i agree with Tynan -- we should have 2 folders assets/public. it's the same thing. Not sure on if it shld be within src or outside src of if that makes any difference.

Reply:
I have just tried extracting those 2 logos out as individual svgs and loading this directly into the `Media` component but no luck - the logos doesn't show up. 

<img width="1438" alt="image" src="https://user-images.githubusercontent.com/27775658/157439347-e2754377-224f-4a41-8b0e-8a4ce47db2c7.png">

@tynandebold Not sure if my setup is wrong, but maybe you can shed some light into this?